### PR TITLE
TST: Add unit tests for PyDMWaveformTable.

### LINKED ISSUE #264: Tests for Waveform Table Widget

### PR DESCRIPTION
**Pull request by hmbui**
STY: Fix spacing in send_wave_form().
Close #264

### CODE REVIEW

File: pydm/widgets/waveformtable.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(self, parent=None, init_channel=None):
     def value_changed(self, new_waveform):
         """
         Callback invoked when the Channel value is changed.
-
+o
```
Comment (hhslepicka) [inline pydm/widgets/waveformtable.py]: Is this a typo?

File: pydm/tests/conftest.py [hunk 2]
```diff
@@ -102,6 +102,7 @@ class ConnectionSignals(QObject):
     prec_signal = pyqtSignal(int)
     upper_ctrl_limit_signal = pyqtSignal([float])
     lower_ctrl_limit_signal = pyqtSignal([float])
+    waveform_signal = pyqtSignal([int, int])
```
Comment (hhslepicka) [inline pydm/tests/conftest.py]: Would you mind to put a comment on this signal so we know that this one is used just at the Waveform Table?

**Comment by hmbui** [comment]
@hhslepicka Fixed the typo and added the comment.

**Comment by hhslepicka** [comment]
@hmbui you have now a conflict somehow... you will need to rebase your branch.

**Comment by hmbui** [comment]
@hhslepicka  Resolved the blank line conflict.

**Pull request closed without merging**